test/fixtures/krb5kdc-fixture/docker-compose.yml

```diff
@@ -8,6 +8,7 @@ services:
     command: "bash /fixture/src/main/resources/provision/peppa.sh"
     volumes:
       - ./build/shared/peppa:/fixture/build
+      - /dev/urandom:/dev/random
```

Comment:
Maybe worth to add a comment as to why this is necessary, it resembles a typo on first glance.